docs/manual.rst

```diff
@@ -1256,13 +1256,39 @@ chart
 
     .. code:: text
 
-        {{% raw %}}{{% chart Bar title='Browser usage evolution (in %)' %}}
-        x_labels='["2002","2003","2004","2005","2006","2007","2008","2009","2010","2011","2012"]'%}}
+        {{% raw %}}{{% chart Bar title='Browser usage evolution (in %)'
+x_labels='["2002","2003","2004","2005","2006","2007"]' %}}
```

Comment:
Is this a syntax change or a fix to the docs?

Reply:
Just a fix for the docs, things were different lengths

Reply:
You moved x_labels from “inside” the shortcode to its arguments.

Reply:
Hmmm that was broken, I think.